project_event/wizard/event_wizard_view.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_project_event_wizard" model="ir.ui.view">
+        <field name="name">Event Wizard</field>
+        <field name="model">project.event.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Event Wizard">
+                <p class="oe_grey">
+                    Event will be created from template in draft so that you can review
+                    the event before validation.
```

Comment:
@rimbendhaou : [Clarification] Not clear what this string really mean for ??????????????